app/routes/_gcn.circulars/route.tsx

```diff
@@ -8,12 +8,15 @@
 import { Outlet } from '@remix-run/react'
 import { GridContainer } from '@trussworks/react-uswds'
 
+import { action as parentAction } from '../_gcn.circulars._archive._index/route'
 import type { BreadcrumbHandle } from '~/root/Title'
 
 export const handle: BreadcrumbHandle = {
   breadcrumb: 'Circulars',
 }
 
+export const action = parentAction
+
```

Comment:
This is weird. Why do we have two routes with the same action?

Reply:
This was a weird one, but without it I was getting an error about an error about the action not being defined. I'll see if I can replicate it

Reply:
Couldn't replicate, removed in latest